src/language-js/printer-estree.js

```diff
@@ -743,7 +743,14 @@ function printPathNoParens(path, options, print, args) {
         ".",
         path.call(print, "property")
       ]);
-    case "BindExpression":
+    case "BindExpression": {
+      const isParentNewExpression =
+        path.getParentNode().type === "NewExpression";
+
+      if (isParentNewExpression) {
+        parts.push("(");
```

Comment:
Is there a reason to do it this way instead of modifying [needs-parens.js](https://github.com/prettier/prettier/blob/2a298a3815890c8f8de4e38f553d3112d8e6fbc2/src/language-js/needs-parens.js)?